README.md

```diff
@@ -17,28 +17,55 @@ Fourier Accelerated Nodal Solvers (FANS) is an FFT-based homogenization solver d
 
 Before proceeding with the installation, ensure that your system has the necessary dependencies. The prerequisites of FANS can be installed using Spack for a streamlined setup on high-performance computing systems, or through traditional package management for personal use.
 
-### Traditional Installation
-
-If you're setting up FANS on a personal computer or in a non-HPC environment, follow these instructions:
-
-Please ensure you have the following dependencies installed on your system:
+The requirements are:
 
-- CMake (version 3.0 or higher)
+- A C++ compiler (e.g. GCC)
+- CMake (version 3.0 or higher) (+ GNU file utility for creating .deb packages)
+- Git (for cloning this repo)
 - MPI (mpicc and mpic++)
 - HDF5 with parallel support
 - Eigen3
 - FFTW3 with MPI support
 
-Specifically, to run FANS, you at least need the following packages:
+### Traditional Installation
```

Comment:
```suggestion
### Installing from source
```

Reply:
This heading was meant to distinguish from the next one 'Spack installation' which is also a from source installation. We have these two from source installation methods and soon when we release packages we will have a prebuilt installation option. Maybe we should change the second level heading to 'Installation from source'?

Reply:
Like so:
## Installation from source
### Prerequisites
### Install prerequisites using apt
### Install prerequisites using Spack
...

Reply:
I think we should be careful while saying *Spack installation*, because by default this means that there is a Spack package for FANS. The use of Spack that we show is only to install the dependencies. There can be different ways to install dependencies, but for FANS itself, we should have one **Installation from source** section. I would suggest segregating the installation of dependencies from the installation of FANS itself.